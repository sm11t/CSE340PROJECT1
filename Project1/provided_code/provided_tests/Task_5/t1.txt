TASKS
    1 5 2
POLY
    T(X, Y) = 2 Y^2 (X^2 + Y)^2;
    S(R, Y, U, W) = R^2 + Y^2 + W^44 + U;
    Y = x + 5555;
EXECUTE
    w = S(2, 6, 3, 1);
    INPUT A;
    INPUT B;
    INPUT C;
    w = Y(A);
    OUTPUT w;
    w = T(A, C);
INPUTS
    9 18 10 7 0 1 2 8 1