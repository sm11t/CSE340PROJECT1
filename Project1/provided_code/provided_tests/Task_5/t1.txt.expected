5564
T: 6
S: 44
Y: 1